app/services/risk_assessment/low_risk_assessment.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RiskAssessment
+  class LowRiskAssessment
+    def initialize(claim)
+      @claim = claim
+    end
+
+    def assess
+      prep_attendance_advocacy? || new_prep_advocacy?
+    end
+
+    def prep_attendance_advocacy?
+      total_prep_time + total_attendance_time <= total_advocacy_time
+    end
+
+    def new_prep_advocacy?
+      prosecution_evidence = (@claim.prosecution_evidence || 0) * 4
+      number_of_pages = (@claim.defence_statement || 0) * 2
+      number_of_witnesses = (@claim.number_of_witnesses || 0) * 30
+      length_of_video = @claim.time_spent || 0
```

Comment:
starting to feel like a lot of magic numbers... might be nicer to have them as constants at the top of the file.

Reply:
Moved to constants